readmes/README.Linux.md

```diff
@@ -66,12 +76,32 @@
   sudo ./README.Ubuntu.bash
   ```
 
+- If the script download fails, you can try changing the software download source for Ubuntu
+
+    ```bash
+    cp /etc/apt/sources.list /etc/apt/sources.list.bak_yyyymmdd
+    vim /etc/apt/sources.list #Open the sources.list file
+    
+    #Add the following content to the end of the file
+    deb http://mirrors.aliyun.com/ubuntu/ bionic main restricted universe multiverse
+    deb http://mirrors.aliyun.com/ubuntu/ bionic-security main restricted universe multiverse
+    deb http://mirrors.aliyun.com/ubuntu/ bionic-updates main restricted universe multiverse
+    deb http://mirrors.aliyun.com/ubuntu/ bionic-proposed main restricted universe multiverse
+    deb http://mirrors.aliyun.com/ubuntu/ bionic-backports main restricted universe multiverse
+    deb-src http://mirrors.aliyun.com/ubuntu/ bionic main restricted universe multiverse
+    deb-src http://mirrors.aliyun.com/ubuntu/ bionic-security main restricted universe multiverse
+    deb-src http://mirrors.aliyun.com/ubuntu/ bionic-updates main restricted universe multiverse
+    deb-src http://mirrors.aliyun.com/ubuntu/ bionic-proposed main restricted universe multiverse
+    deb-src http://mirrors.aliyun.com/ubuntu/ bionic-backports main restricted universe multiverse
```

Comment:
This is not global. We just recommend users to try another one Ubuntu software source?